src/lib/data-root.ts

```diff
@@ -160,7 +160,7 @@ export class DataRootComputer {
     });
   }
 
-  async computeDataRoot(id: string): Promise<Uint8Array | undefined> {
+  async computeDataRoot(id: string): Promise<string | undefined> {
```

Comment:
@hlolli - the result of this is the `toB64Url` of the data root, lmk if i'm mistaken on this change. it was causing type warnings in the `verifyDataRoot` when trying to compare a `string` to `Uint8Array`

Reply:
I think it was converted later, looking...

Reply:
that is correct, initially it was that, but after my changes it's a string